supertokens_python/recipe/emailverification/emaildelivery/service/smtp/implementation.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
```

Comment:
rename this file to service_implementation.py

Reply:
Done.